s2i/bin/run

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+source /opt/app-root/etc/generate_container_user
+
+set -e
+
+. ${VARNISH_CONFIGURATION_PATH}/varnish.params
+
+if grep "NAME=Fedora" /etc/os-release; then
```

Comment:
Why Fedora differs?
(I don't know varnish much, so that is why I'm asking)

Reply:
(fyi, me neither)

The CLI options of varnishd were changed significantly between version 4 and 5. No backwards compatibility. Pretty nuts.

Reply:
Thanks for explanation

Reply:
I was thinking about this more.

Varnish 4.1 changed that parameters which differs. It is not about Fedora but it will probably change even for newer varnish in some SCL. So it would be better to base the params on varnishd version.